tripper/mappings/mappings.py

```diff
@@ -9,15 +9,14 @@
 with numpy.
 
 """
-# pylint: disable=invalid-name,redefined-builtin
+# pylint: disable=invalid-name,redefined-builtin, too-many-lines
 from __future__ import annotations  # Support Python 3.7 (PEP 585)
 
 import subprocess  # nosec: B404
 from collections import defaultdict
 from enum import Enum
 from typing import TYPE_CHECKING, Mapping, Sequence
 
-import numpy as np
 from pint import Quantity  # remove
```

Comment:
This is not part of this PR, so this comment can be disregarded (or a follow-up issue opened).

Why is there a `# remove` comment for this line?

Reply:
Don't know. I'd leave it for now.

Reply:
made issue